drivers/serial/uart_ns16550.c

```diff
@@ -421,6 +439,13 @@ static int uart_ns16550_configure(const struct device *dev,
 	}
 #endif
 
+#if CONFIG_UART_NS16550_TI_K3
```

Comment:
We usually do `#ifdef` or `#if defined()` for boolean type Kconfigs.

Reply:
Have made the change.